### ISSUE TITLE
classify_webcam.py on Mac runs but no windows open

### ISSUE THREAD

**Issue by DennisFaucher** [loicmarie/sign-language-alphabet-recognizer#17]
train.py and classify.py run fine. classify_webcam.py doesn't do anything. Eventually, python3 shows as unresponsive in Activity Monitor.  CTRL-C break sends this output:

Traceback (most recent call last):
  File "classify_webcam.py", line 57, in <module>
    ret, img = cap.read()
TIA.

**Comment by someshium** [comment]
@DennisFaucher Did you got the solution for this problem. the classify_webcam.py doesn't works for me. it gets struck for infinity. i am using ubuntu

**Comment by DennisFaucher** [comment]
No. I built my own. See this repo: https://github.com/DennisFaucher/pytorchasl

**Comment by saugaatallabadi** [comment]
import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()
